packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- `Theme`: Remove "experimental" designation ([#61077](https://github.com/WordPress/gutenberg/pull/61077)).
+
```

Comment:
Can remove, as in https://github.com/WordPress/gutenberg/pull/61078/files#r1580009601